crates/ide_assists/src/handlers/wrap_return_type_in_result.rs

```diff
@@ -98,10 +105,17 @@ mod tests {
 
     use super::*;
 
-    #[test]
-    fn wrap_return_type_in_result_simple() {
+    fn check(ra_fixture_before: &str, ra_fixture_after: &str) {
         check_assist(
             wrap_return_type_in_result,
+            &format!("//- minicore: result\n{}", ra_fixture_before.trim_start()),
```

Comment:
I am not a huge fan of hiding meta lines in the check function. Repeating `//- minicore:result` everywhere is formal duplication, but semantically, it is not. You fundamentally need to signal, in each test, whether it uses minicore or not.

By extracting this into a function, the decision becomes hidden.

Reply:
Fair point, I'll correct that then.